renderer/components/Gamepad.tsx

```diff
@@ -40,8 +40,8 @@ export class Gamepad extends React.Component<Props, State> {
   roundedValues = () => {
     const gamepadButtons: string[] = [];
 
-    for (let i = 0; i < 32; i++) {
-      gamepadButtons.push(numeral((this.props.gamepad.buttons & (1 << i)) >> i).format('0'));
+    for (let i = 0; i < NUM_GAMEPAD_BUTTONS; i++) {
+      gamepadButtons.push(numeral((Number(this.props.gamepad.buttons) & (1 << i)) >> i).format('0'));
```

Comment:
Just to be sure, we cast to `Number` here now because Input could be a long also now right?

Reply:
Yea, it can be a long but for gamepad we know its less than 32 bits so we can cast it to a Number to do bit shift operations.